### PR TITLE
feat(APIv2): RHINENG-11779 expose os_versions on systems/SGs

### PR DESCRIPTION
**Pull request by skateman**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: app/policies/v2/system_policy.rb [hunk 1]
```diff
@@ -23,6 +23,10 @@ def destroy?
       match_account? && match_group?
     end
 
+    def os_versions?
+      true
```
Comment (romanblanco) [inline app/policies/v2/system_policy.rb:27]:
```suggestion
      match_account? && match_group?
```

Comment (skateman) [reply app/policies/v2/system_policy.rb:27]: No, as this is applied on a collection and there you don't have individual systems to match against, see `#index`

**Review by romanblanco — APPROVED** [review]
LGTM :+1:

**Pull request merged**